test/services/FootprintService.test.js

```diff
@@ -149,12 +149,7 @@ describe('api.services.FootprintService', () => {
         .then(role => {
           assert(role)
           assert(role.id)
-          return FootprintService.find('User', userId)
-        })
-        .then(user => {
-          assert(user[0])
-          assert.equal(user[0].roles.length, 1)
-          assert.equal(user[0].roles[0].name, 'createassociatedrole')
```

Comment:
You're right ! That was my old test witch can be rewrite from scratch if you want ^^ you should retrieve here the user and his children and test it was created correctly